questions/largeInput.tsx

```diff
@@ -0,0 +1,68 @@
+import React from 'react';
+import { StyleSheet, Text, View } from 'react-native';
+import { TextInput } from 'react-native-paper';
+
+interface largeTextProps {
+    displayText: string;
+    description?: string;
+}
+
+const styles = StyleSheet.create({
+    container: {
+        display: 'flex',
+        alignItems: 'center',
+        marginBottom: 24
+    },
+    displayText: {
+        width: '74%',
+        height: 17,
+        fontWeight: '600',
+        fontSize: 14,
+        lineHeight: 17,
+        color: '#2B2B2B',
+        marginBottom: 8
+    },
+    description: {
+        width: '74%',
+        fontSize: 14,
+        lineHeight: 17,
+        color: '#6A6A6A',
+        marginBottom: 8
+    },
+    input: {
+        width: '74%',
+        height: 116,
+        fontSize: 12,
+        lineHeight: 14,
+        color: '#6A6A6A',
+        borderWidth: 1,
+        borderColor: '#2B2B2B',
+        borderRadius: 8,
+        backgroundColor: '#FFFFFF'
+    }
+});
+
+/*
+function renderLargerText(props: largeTextProps) {
```

Comment:
Can you enable this function across all questions, and pass in dummy data from HomeScreen.tsx? This means no coded content in any of the components.